normandy/control/static/control/js/components/action_forms/ConsoleLogForm.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+class ConsoleLogForm extends React.Component {
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    const { fields } = this.props;
+    return (
+      <div className="row">
+        {
+          Object.keys(fields).map(fieldName => {
```

Comment:
This feels really weird. I get why: If we want to depend on the argument schema for defining these forms, we need to do this to handle new fields being added.